raiden/tests/integration/fixtures/smartcontracts.py

```diff
@@ -108,6 +137,15 @@ def secret_registry_proxy(deploy_client, secret_registry_address, contract_manag
     )
 
 
+@pytest.fixture
+def service_registry_proxy(deploy_client, service_registry_address, contract_manager):
+    return ServiceRegistry(
+        jsonrpc_client=deploy_client,
+        service_registry_address=to_canonical_address(service_registry_address),
```

Comment:
IIUC `service_registry_address` can be `None` with will cause an exception in `to_canonical_address`.